### PR TITLE
Update AddEventCommandParser and DeleteEventCommandParser

### LINKED ISSUE #52: Update Parser: AddEventCommandParser

### LINKED ISSUE #53: Update Parser: DeleteEventCommandParser

### PR DESCRIPTION
**Pull request by YuFeng0930**
What have been done:
- Add AddEventCommandParser class and DeleteEventCommandParser class without deleting the AB3 AddCommandParser and DeleteCommandParser.
- The parsers' dependencies are with SocheduleParserUtil
- Restore the AboutUs table (which was delete by me last time).
Closes #52, closes #53.

### CODE REVIEW

**Review by ljhgabe — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/AddEventCommandParser.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                || !argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddEventCommand.MESSAGE_USAGE));
+        }
+
+        Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
+        Date startDate = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_STARTDATE).get());
+        Time startTime = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_STARTTIME).get());
+        Date endDate = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_ENDDATE).get());
+        Time endTime = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_ENDTIME).get());
+        Set<Category> categoryList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_CATEGORY));
```
Comment (ljhgabe) [inline src/main/java/seedu/address/logic/parser/AddEventCommandParser.java]: Maybe can change the parseTags here to a parseCategories? 🤔

Comment (YuFeng0930) [reply src/main/java/seedu/address/logic/parser/AddEventCommandParser.java]: Thanks for spotting the error:)

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -11,12 +11,12 @@
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.common.Category;
 import seedu.address.model.common.Date;
+import seedu.address.model.common.Name;
```
Comment (litone01) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: this should not be imported yet.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 3]
```diff
@@ -11,12 +11,12 @@
...
+import seedu.address.model.common.Tag;
```
Comment (litone01) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: this should not be imported yet.

File: src/main/java/seedu/address/logic/commands/AddEventCommand.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_PHONE + "PHONE "
+            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "John Doe "
+            + PREFIX_PHONE + "98765432 "
+            + PREFIX_EMAIL + "johnd@example.com "
```
Comment (litone01) [inline src/main/java/seedu/address/logic/commands/AddEventCommand.java:31]: are we gonna change the prefix_name and phone and email etc

**Pull request merged**